src/contexts/Polkawatch/index.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
```

Comment:
Perhaps this `Polkawatch` folder can sit inside the `Plugins` folder, so we have them all under one umbrella. So `src/contexts/Plugins/Polkawatch/index.tsx`.

Reply:
Implemented